snippets/card-product.liquid

```diff
@@ -495,8 +495,17 @@
                     </div>
                     <div>
                       <div class="quick-add-modal__content-info--bulk-details small-hide medium-hide">
-                        <h3 class="h2">{{ card_product.title | escape }}</h3>
-                        {% render 'price', product: card_product, price_class: '', show_compare_at_price: true %}
+                        <a
+                          href="{{ card_product.url }}"
+                          id="StandardCardNoMediaLink-{{ section_id }}-{{ card_product.id }}"
+                          class="full-unstyled-link"
+                          aria-labelledby="StandardCardNoMediaLink-{{ section_id }}-{{ card_product.id }} NoMediaStandardBadge-{{ section_id }}-{{ card_product.id }}"
+                        >
+                          <h3 class="h2">
+                            {{ card_product.title | escape }}
+                          </h3>
+                          {% render 'price', product: card_product, price_class: '', show_compare_at_price: true %}
+                        </a>
```

Comment:
I think here we could move the link to wrap the `{{ card_product.title | escape }}` only rather than the h3 element as well as the price.

And the ID could be simpler? Or why does it mention `NoMedia` ?

Reply:
On the standard quick add, only the product title is linked. Not the price part. So I think ill lean towards title only 👍

Reply:
Please just allow the product title to link to the product. It may feel natural on a product card due to the nature of that target space. Just the title is intuitive since it also gets the underline. This is matching how we do it in Cart. Thanks!
![30-16-sllmr-x597f](https://github.com/Shopify/dawn/assets/26230969/907e7591-b8ec-43a3-8031-fa8b608b46f5)